### ISSUE TITLE
Try using an array for outputbuffer in DrawClippedLine()

### ISSUE THREAD

**Issue by objorke** [oxyplot/oxyplot#127]
**Imported from [http://oxyplot.codeplex.com/workitem/10174](http://oxyplot.codeplex.com/workitem/10174)**

**[objo](http://www.codeplex.com/site/users/view/objo)** 2014-04-06 09:41
Try using an array instead of a list for the outputbuffer in the DrawClippedLine extension method.

See also discussion in <a href="https://oxyplot.codeplex.com/workitem/10170" rel="nofollow">
https://oxyplot.codeplex.com/workitem/10170</a>

**Comment by tibel** [comment]
For me there are three key areas of performance issues (without going into rendering details):
- ItemsSource (data binding)
- Smoothing (LineSeries Smooth=true)
- OxyImage (e.g. Heatmap)

**Comment by objorke** [comment]
I agree these are areas that should be profiled and improved. Each of those should be a separate issue, to make them easier to close. 
I think this issue that was raised in the codeplex discussion is a very specific implementation detail, and probably not very difficult to solve.

**Comment by tibel** [comment]
You are right. I have created separate issues for the above points.
